command.go

```diff
@@ -923,9 +938,12 @@ func (c *Command) execute(a []string) (err error) {
 	}
 
 	if !c.Runnable() {
-		return flag.ErrHelp
+		if c.ErrorOnUnknownSubcommand {
```

Comment:
`c` is the current command here.  Therefore I would need to set this new flag for every command throughout my program to get the new behaviour globally.

Do you feel this feature should be enabled per command or globally?

If globally, then you can add a global variable in `cobra.go` instead of in the command structure.  Then the program can enable this feature in a single location.

Reply:
Hey there! You are right. I think the global behavior is more what I (and probably others) want! I will change that!

Reply:
Hi @marckhouzam. Did you have a chance to review my changes? Would the behavior be so ok?